app/views/common_parameters/welcome.html.erb

```diff
@@ -1,12 +1,13 @@
+
 <% content_for(:title, _("Global Parameters")) %>
-<div class="blank-slate-pf">
-  <div class="blank-slate-pf-icon">
-    <%= icon_text("list-ul", "", :kind => "fa") %>
-  </div>
-  <h1><%= _('Global Parameters') %></h1>
-  <p><%= _("Foreman allows to define a hierarchy of parameter inheritance, where global parameters accessible from any manifest.") %></p>
-  <p><%= link_to _('Learn more about this in the documentation.'), documentation_url("global-parameters")%></p>
-  <div class="blank-slate-pf-main-action">
-    <%= new_link(_("Create Global Parameter"), :class => 'btn-lg') %>
```

Comment:
Note that `new_link` checks if the user is permitted to create the resource before showing the link - https://github.com/theforeman/foreman/blob/develop/app/helpers/application_helper.rb#L145
So when switching this to use react, we should only pass the `create` action if the user has the correct permission (or they will get a link that they cannot use). 
This is true for many of the other pages as well. For the custom components, you can add a prop that defines if the user can create and decide according to that if the link should be displayed.